cmd/api/rpc/comment.go

```diff
@@ -0,0 +1,56 @@
+package rpc
+
+import (
+	"context"
+	"fmt"
+	"github.com/Tiktok-Lite/kotkit/kitex_gen/comment"
+	"github.com/Tiktok-Lite/kotkit/kitex_gen/comment/commentservice"
+	"github.com/Tiktok-Lite/kotkit/pkg/etcd"
+	"github.com/Tiktok-Lite/kotkit/pkg/helper/constant"
+	"github.com/cloudwego/kitex/client"
+	"github.com/spf13/viper"
+)
+
+var (
+	commentClient commentservice.Client
+)
+
+func InitComment(config *viper.Viper) {
+	r, err := etcd.Resolver()
+	if err != nil {
+		logger.Errorf("Error occurs when creating etcd resolver: %v", err)
+		panic(err)
+	}
+
+	commentServiceName := config.GetString("server.name")
+
+	c, err := commentservice.NewClient(commentServiceName, client.WithResolver(r))
+
+	if err != nil {
+		logger.Errorf("Error occurs when creating login client: %v", err)
+		panic(err)
+	}
+	commentClient = c
+}
+
+func CommentList(ctx context.Context, req *comment.DouyinCommentListRequest) (*comment.DouyinCommentListResponse, error) {
+	resp, err := commentClient.CommentList(ctx, req)
+	if err != nil {
+		return nil, err
```

Comment:
```suggestion
		return resp, err
```
这样做的好处是可以在http端处理到错误，（虽然这个几乎永远不会触发）